docs/rules/explicit-length-check.md

```diff
@@ -35,13 +35,13 @@ if (string.length < 1) {}
 ### Pass
 
 ```js
-if (array.length !== 0) {}
+if (array.length === 0) {}
```

Comment:
I believe this was a typo because this section talks about zero comparisons. So I fixed it.